AutoCollection/ClientRequestParser.ts

```diff
@@ -48,15 +53,26 @@ class ClientRequestParser extends RequestParser {
         let dependencyName = this.method.toUpperCase() + " " + url.format(urlObject);
 
         let remoteDependency = new ContractsModule.Contracts.RemoteDependencyData();
-        remoteDependency.target = urlObject.hostname;
+
+        if (this.targetIKeyHash) {
+            // Currently the service supports a dependency type name of ApplicationInsights,
+            // but not a dependency kind. So the dependency kind here is still just HTTP.
+            remoteDependency.dependencyKind = ContractsModule.Contracts.DependencyKind.Http;
+            remoteDependency.dependencyTypeName = ContractsModule.Contracts.DependencyKind
+                [ContractsModule.Contracts.DependencyKind.ApplicationInsights];
```

Comment:
Instead of the string value 'ApplicationInsights', we have updated it to 'Application Insights'. This is more in line with the rest of the known types like "Azure Blobs" etc. 
See: https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/nizarq/Src/DependencyCollector/Shared/Implementation/RemoteDependencyConstants.cs